nap/utils.py

```diff
@@ -42,13 +41,20 @@ def flatten_params(k, vs):
 
         flat_params = [
             flatten_params(k, v)
-            for (k, v) in params.iteritems()
+            for (k, v) in params.items()
         ]
 
         # since we can have more than one value for a single key, we use a
         # tuple of two tuples instead of a dictionary
         params_tuple = tuple(itertools.chain(*flat_params))
-        param_string = urllib.urlencode(params_tuple)
+        param_string = urllib.parse.urlencode(params_tuple)
```

Comment:
I take it back, this is fine. See the import above:

from six.moves import urllib

so it will work correctly in py2 and py3.